cypress/e2e/tests/upgrade/create_upgrade_data.test.ts

```diff
@@ -155,5 +160,12 @@ describe(["@pre-upgrade"], "Creating pre-requisites before an upgrade", () => {
         );
         binaryApplication.analyze();
         binaryApplication.verifyAnalysisStatus("Completed");
+        binaryApplication.selectApplicationRow();
+    });
+
+    it("Creating assessment", function () {
```

Comment:
Maybe "Assess application" is a better fit for this test's name

Reply:
Chenged